tests/testVisitInfo.py

```diff
@@ -195,10 +195,10 @@ def testSetVisitInfoMetadata(self):
 
     def testSetVisitInfoMetadataMissingValues(self):
         """If a value is unknown then it should not be written to the metadata"""
-        visitInfo = afwImage.makeVisitInfo()  # only rot type is known
+        visitInfo = afwImage.VisitInfo()  # only rot type is known
```

Comment:
We get the same warnings with SWIG now. And if someone decides to fix them in master we get merge conflicts. Better alternative would be to fix in master instead of here and then pull it in with the next rebase.